src/utils/selectors.ts

```diff
@@ -161,8 +161,8 @@ export const venueSelector = (state: RootState) =>
     : undefined;
 
 export const parentVenueOrderedSelector: SparkleSelector<
-  WithId<AnyVenue> | undefined
-> = (state) => state.firestore.ordered.parentVenue?.[0];
```

Comment:
It was only used in `useRelatedVenues`. I changed the logic there as well

Reply:
Makes sense.

For future reference, this seems to have been changed so that we can use `isLoaded` as per the following:

```typescript
const parentVenueRaw = useSelector(parentVenueOrderedSelector);
const parentVenue = parentVenueRaw?.[0];

const isParentVenueLoaded = isLoaded(parentVenueRaw);
```

I believe the way it used to be meant that we wouldn't have been able to use `isLoaded`.

Reply:
Just for my own knowledge/reference of how `isLoaded` works..

Looking at the definition of `isLoaded`, it seems to check that none of the arguments passed to it are `undefined`, and that lodash's `get(arg, 'isLoaded') !== false`:

- https://github.com/prescottprue/react-redux-firebase/blob/e67bbd71ab25fc4830d881431a1661a93b316efc/src/helpers.js#L83-L127

```typescript
import {
// ..snip..
  get,
// ..snip..
} from 'lodash'

// ..snip..

export function isLoaded(...args) {
  return !args || !args.length
    ? true
    : args.every((arg) => arg !== undefined && get(arg, 'isLoaded') !== false)
}
```

- https://lodash.com/docs/4.17.15#get
  - > Gets the value at path of object. If the resolved value is undefined, the defaultValue is returned in its place.